measCompApp/src/drvMultiFunction.cpp

```diff
@@ -817,7 +833,7 @@ MultiFunction::MultiFunction(const char *portName, const char *uniqueID, int max
       ASYN_MULTIDEVICE | ASYN_CANBLOCK, 1, /* ASYN_CANBLOCK=1, ASYN_MULTIDEVICE=1, autoConnect=1 */
       0, 0),  /* Default priority and stack size */
     pollTime_(DEFAULT_POLL_TIME),
-    forceCallback_(1),
+    forceCallback_{1,1,1},
```

Comment:
This initialization should probably be done in the constructor using a loop over MAX_IO_PORTS so it changes automatically when that number is increased.

Reply:
ok, moved this initialization into the constructor